xml/System.Diagnostics.CodeAnalysis/FeatureGuardAttribute.xml

```diff
@@ -29,14 +29,35 @@
   </Attributes>
   <Docs>
     <summary>
-      <para>Indicates that the specified public static boolean get-only property</para>
-      <para>guards access to the specified feature.</para>
+      <para>
+        Indicates that the specified public static boolean get-only property
```

Comment:
```suggestion
        Indicates that the specified public static Boolean read-only property
```

Reply:
No. Eventually I hope to reconcile all the comments, but it's a big project and it won't happen for a while. Once we do, though, then we won't allow edits in this repo anymore.